src/FixedDataTableRow.js

```diff
@@ -259,16 +239,13 @@ class FixedDataTableRow extends React.Component {
       'fixedDataTableRowLayout/main': true,
       'public/fixedDataTableRow/main': true,
       'public/fixedDataTableRow/highlighted': this.props.index % 2 === 1,
-      // 'public/fixedDataTableRow/odd': this.props.index % 2 === 1,
-      // 'public/fixedDataTableRow/even': this.props.index % 2 === 0,
-      //No CSS corresponding to these (archit)
+      'public/fixedDataTableRow/odd': this.props.index % 2 === 1,
+      'public/fixedDataTableRow/even': this.props.index % 2 === 0,
     });
     var fixedColumnsWidth = this.props.fixedColumnsWidth;
     var fixedRightColumnsWidth = this.props.fixedRightColumnsWidth;
     var scrollableColumnsWidth = this.props.scrollableColumnsWidth;
-    // var Archit=(
-    //   <
-    // )
+
     var fixedColumns = (
       <FixedDataTableCellGroup
```

Comment:
Can we remove this FixedDataTableCellGroup component at all ?,   I think we may directly render cell from row.
Please investigate this as well and see if it may lead to any performance improvement.

Bonus, if we could also remove FixedDataTableCell and just directly render cells supplied by client.